apps/blockchain/test/blockchain_test.exs

```diff
@@ -2,1237 +2,434 @@ defmodule BlockchainTest do
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d16g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d17g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d18g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d19g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d1g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d20g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d21g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d22g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d23g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d24g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d25g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d26g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d27g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d28g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d29g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d2g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d30g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d31g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d32g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d34g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d35g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d36g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d37g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d38g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d39g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d3g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d40g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d41g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d42g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d43g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d44g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d45g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d46g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d47g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d48g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d49g0v0.json",
-    "GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d4g0v0.js
```

Comment:
won't this skip testing on all forks if one of the forks has a failing test?

Reply:
@masonforest  yes, but the failing tests should be described in the module attribute